modules/apps/frontend-taglib/frontend-taglib-clay/package.json

```diff
@@ -16,7 +16,6 @@
 		"@clayui/form": "3.8.0",
 		"@clayui/icon": "3.0.5",
 		"@clayui/label": "3.3.1",
-		"@clayui/layout": "3.1.0",
```

Comment:
You can't remove this from here. This is actually the one place where you can't, because this is where everyone else consumes it from.


Sorry about that 😂

Reply:
🤔 maybe we should insert a comment into the file

> 💀🚨 DANGER! DEATH! DO NOT REMOVE THESE DEPENDENCIES! 💀🚨